languages/csharp/exercises/concept/dates/.docs/instructions.md

```diff
@@ -44,3 +44,12 @@ Implement the `Appointment.Description` method that takes an appointment date an
 Appointment.Description(new DateTime(2019, 03, 29, 15, 0, 0))
 // => "You have an appointment on Friday 29 March 2019 at 15:00."
 ```
+
+### 5. Return the anniversary date
+
+Implement the `Appointment.AnniversaryDate` method that returns this year's anniversary date, which is September 15th:
+
+```csharp
+Appointment.AnniversaryDate()
```

Comment:
Not sure of the convention in use here, but should there be semicolons at the end of these statements?
```suggestion
Appointment.AnniversaryDate();
```

Reply:
I am not sure which is preferrable. There is something to be said for both. The way I saw it was that it was less like a statement, but more like an expression being evaluated (for example in the C# interactive). Also, the `;` was another thing for the student to process, and I was trying to focus on just calling the method. I can see an argument for the `;` to be made though.

@robkeim what do you think?

Reply:
I don't have a strong preference for one way over another, but I thought it was slightly more logical without the semicolon at the end because the sentences are talking about methods that you need to implement and the implementation doesn't take a semicolon, but an open brace to start the implementation.